precompile/params.go

```diff
@@ -9,8 +9,10 @@ import "github.com/ethereum/go-ethereum/common"
 const (
 	ModifyAllowListGasCost = 20_000
 	ReadAllowListGasCost   = 5_000
-
-	MintGasCost = 30_000
+	MintGasCost            = 30_000
+	// TODO: decide on gas costs
+	SetFeeConfigGasCost = 30_000
```

Comment:
Can we increase the gas cost to match 20k per state slot that we need to write to perform this function?

Reply:
Same comment for read to base this off of the number of state slots that we need to read to fulfill the request